docs/rules/0136/response-message-name.md

```diff
@@ -0,0 +1,159 @@
+---
+rule:
+  aip: 136
+  name: [core, '0136', response-message-name]
+  summary: Custom methods must have standardized response message names.
+permalink: /136/response-message-name
+redirect_from:
+  - /0136/response-message-name
+---
+
+# Custom methods: Response message
+
+This rule enforces that all custom methods should take a response message
+matching the RPC name, with a `Response` suffix, or the resource being operated
+on [AIP-136][].
```

Comment:
```suggestion
This rule enforces that custom methods have a response message that is named to
match the RPC name with a `Response` suffix, or that is the resource being operated
on, as described in [AIP-136][].
```

Just wanted to tighten it up a bit, take the "should" out of it, the guidance on AIP-136 will dictate the requirement level.